### PR TITLE
ocpqe 15876-20230906

### PR DESCRIPTION
**Pull request by lenahorsley**
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

### CODE REVIEW

File: ci-operator/step-registry/openshift-qe/ocp-qe-perfscale-ci-tests/openshift-qe-ocp-qe-perfscale-ci-tests-chain.yaml
Comment (shahsahil264) [inline ci-operator/step-registry/openshift-qe/ocp-qe-perfscale-ci-tests/openshift-qe-ocp-qe-perfscale-ci-tests-chain.yaml:1]: Hey @lenahorsley , I would suggest to put the concurrent build at the top of chain for now, while you are testing it, as you will have to wait and retry a lot, if any of the other tests before concurrent build fail

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by lenahorsley** [comment]
Closing PR

**Pull request closed without merging**